runtime/gc_base/GCExtensions.cpp

```diff
@@ -311,3 +311,30 @@ MM_GCExtensions::releaseNativesForContinuationObject(MM_EnvironmentBase* env, j9
 	}
 #endif /* JAVA_SPEC_VERSION >= 19 */
 }
+
+bool
+MM_GCExtensions::needScanStacksForContinuationObject(J9VMThread *vmThread, j9object_t objectPtr)
+{
+	bool needScan = false;
+#if JAVA_SPEC_VERSION >= 19
+	jboolean started = J9VMJDKINTERNALVMCONTINUATION_STARTED(vmThread, objectPtr);
+	jboolean finished = J9VMJDKINTERNALVMCONTINUATION_FINISHED(vmThread, objectPtr);
+	J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(vmThread, objectPtr);
+	/**
+	 * We don't scan mounted continuations:
+	 *
+	 * for concurrent GCs, since stack is actively changing. Instead, we scan them during preMount or during root scanning if already mounted at cycle start or during postUnmount (might be indirectly via card cleaning) or during final STW (via root re-scan) if still mounted at cycle end
+	 * for sliding compacts to avoid double slot fixups
+	 *
+	 * For fully STW GCs, there is no harm to scan them, but it's a waste of time since they are scanned during root scanning already.
+	 *
+	 * We don't scan currently scanned either - one scan is enough.
+	 * we don't scan the continuation object before started and after finished - java stack does not exist.
+	 */
+	if (started && !finished) {
+		Assert_MM_true(NULL != continuation);
+		needScan = !isContinuationMountedOrConcurrentlyScanned(continuation);
```

Comment:
I'm guessing this won't compile (needs VM_Helpers:: scope, but then it begs a question should more helpers move, as well.
Another dilemma is where we put all of this. For APIs that are used only by GC, this file/class might be slightly better than generic VMHelpers, but possible even better is ContinuationHelpers?